src/ploomber_core/telemetry/telemetry.py

```diff
@@ -66,10 +66,11 @@ class UserSettings(Config):
     cloud_key: str = None
     user_email: str = None
     stats_enabled: bool = True
+    writable: bool = True
```

Comment:
do we need to add this property here? this will add a new property to the stored file, which users can modify. but we only need this for internal purposes, so it looks like we're exposing internal stuff. if this is a subclass of Config, can't we use the existing property declared in the subclass?

Reply:
So I see you added writable to Config, that's fine.

However, you've also added it to all the subclasses (e.g., Internal). My point is that we should not add it as an annotation to the subclasses because all annotations are persistent to the YAML file (you can see the Config implementation and you'll see that __annotations__ is stored in the YAML file) because it's exposing internals of the implementation.

So the end goal is: Config should manage its internal state transparently. subclasses of config should no need any modifications because that's the point of making them subclasses.

What happens if we delete the annotation? if Config defines writable, subclasses should have access to it, right?

Also, let's rename it to `_writable` to indicate that this is an internal method, not part of the public API.